internal/provider/requestable_permission_resource.go

```diff
@@ -69,7 +69,7 @@ func (r *requestablePermissionResource) Schema(ctx context.Context, req resource
 				},
 			},
 			"app_id": schema.StringAttribute{
-				Required:            true,
+				Computed:            true,
```

Comment:
Is this true? I wouldn't think we would need to change anything on the `requestable_permission` resource itself

Reply:
fixed!